internal/backends/gl/build.rs

```diff
@@ -0,0 +1,10 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
```

Comment:
`cargo xtask check_license_headers` doesn't allow other valid copyright headers. This is problematic if you vendor third party code. Generally this wouldn't be a big deal with Rust because you'd typically use a crate via Cargo instead of vendoring code.